protocol.md

```diff
@@ -1836,6 +1845,27 @@ interface CompletionItem {
 	 */
 	insertTextFormat?: InsertTextFormat;
 	/**
+	 * Whether automatic relative indentation fixes are applied to a multi-line insert text.
+	 * This option applies to both the `insertText` property and the `newText` property
+	 * of a provided `textEdit`.
+	 *
+	 * Relative indentation fixes are applied as follows:
+	 *
+	 * If the insert text contains no line breaks, no fixes are applied.
+	 *
+	 * Otherwise, define the reference line as the line in document where the insert text
+	 * is being applied.
+	 *
+	 * The reference indent is determined by taking all the whitespace
+	 * characters from the front of the reference line upto either the first non-whitespace
+	 * character; or the start position where the insert text is being applied. Whichever
+	 * of these two yields the shortest string is taken as the reference indent.
+	 *
+	 * To adjust indentation, the reference indent is then inserted into the insert text right
+	 * after every line break.
+	 */
+	relativeIndent?: boolean
```

Comment:
I'd assume that leaving this out will default to `relativeIndent = false`, but that is not the current behavior of vscode. So either we tunr the property name around (e.g. `disableRelativeIndent`) or it is stated that an undefined property means `true` (which sounds weird).